engine/class_modules/warlock/sc_warlock_demonology.cpp

```diff
@@ -1121,16 +1121,16 @@ namespace warlock {
     def->add_action( "call_action_list,name=build_a_shard" );
 
     np->add_action("call_action_list,name=nether_portal_building,if=cooldown.nether_portal.remains<20");
-    np->add_action("call_action_list,name=nether_portal_active,if=cooldown.nether_portal.remains>160");
+    np->add_action("call_action_list,name=nether_portal_active,if=cooldown.nether_portal.remains>165");
```

Comment:
APL stayed in the Nether Portal Active action list for 20s instead of the actual 15s